CHANGELOG.md

```diff
@@ -112,17 +133,17 @@ shout-out to @marbar3778 for diagnosing it quickly.
 
 ## v0.34.6
 
-*February 18, 2021* 
+*February 18, 2021*
 
 _Tendermint Core v0.34.5 and v0.34.6 have been recalled due to release tooling problems._
 
 ## v0.34.4
 
 *February 11, 2021*
 
-This release includes a fix for a memory leak in the evidence reactor (see #6068, below). 
-All Tendermint clients are recommended to upgrade. 
-Thank you to our friends at Crypto.com for the initial report of this memory leak! 
+This release includes a fix for a memory leak in the evidence reactor (see #6068, below).
```

Comment:
Should we also re-run the linkifier? Or maybe that can be a separate cleanup.

Reply:
Ah, good call! I'll just do that in a separate PR for master.